library/dicom_frame.php

```diff
@@ -44,6 +44,7 @@
 $twig = (new TwigContainer(null, $GLOBALS['kernel']))->getTwig();
 echo $twig->render("dicom/dicom-viewer.html.twig", [
     'assets_static_relative' => $GLOBALS['assets_static_relative']
+    ,'web_root' => $web_root
```

Comment:
I was scratching my head thinking what broke, but apparently the bug was that the twig file expects **web_root** when the global twig value defined here is **webroot** https://github.com/openemr/openemr/blob/3a1343e3861caf760d199e8385f17f26419d5d13/src/Common/Twig/TwigExtension.php#L72

Ouch, my bad.

Reply:
So this solves the problem but I should have used webroot when I did the refactor.  Keep forgetting to check the webroot extension when doing twig refactors.

Reply:
If this is all from a refactor to twig i'd say you got off pretty good! :)